### ISSUE TITLE
Where the Loss function is indicated for the training phase? 

### ISSUE THREAD

**Issue by luigidamico100** [open-mmlab/mmaction2#1354]
Hello guys! 
I need to know which loss function is used for the training phase but I really can't find the line of code where it is inserted. 
I think it's kind embedded in the model inside but I am not so used with that. I just need to know what loss function is used for the training and eventually change it. Many thanks!

**Comment by dreamerlin** [comment]
1 [Add new loss](https://github.com/open-mmlab/mmaction2/blob/master/docs/en/tutorials/5_new_modules.md#add-new-loss)
2. [loss-related files](https://github.com/open-mmlab/mmaction2/tree/master/mmaction/models/losses)

**Comment by luigidamico100** [comment]
My point was just to know where to find the loss function used for the training I have done and, eventually, changing it. I finally find my answer. 
The loss function is passed by the model definition in the folder `configs/_base_/models/ ` to the head module in `mmaction/models/heads/`. 
The loss is thus computed inside the head module. Correct me if I am wrong!

Thank you so much 👍
